tests/pytest_runner.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+import pytest
+
+from yt.config import ytcfg
+
+if __name__ == "__main__":
+    os.environ["OMP_NUM_THREADS"] = "1"
+    pytest_args = [
+        f"--local-dir={os.path.join(ytcfg.get('yt', 'test_data_dir'), 'answers')}",
+        "-c=pytest_answer.ini",
+        "--junitxml=unittests.xml",
+        "--answer-big-data",
+        f"-n {int(os.environ.get('NUM_WORKERS', 6))}",
```

Comment:
why 6 workers ?
How does pytest react when we ask for more workers than the machine allows ?

Reply:
Cause it's a nice even number that's greater than 1 and less than infinity ;-) if you want to dive even deeper in the root cause of that: I was trained to use 1.5 * n, where n is number of cores available and my laptop happens to have 4 of them. I don't want to make this decision less significant, but this file will only ever be used on Jenkins, which happens to set NUM_WORKERS variable to 12. 
As for the 2nd question I think you're confusing number of processes with number of available processors. I seriously doubt anyone will ever run `pytest -n $(cat /proc/sys/kernel/pid_max)` or `pytest -n $(ulimit -u)`. However, I must shamefully admit I have no idea what will happen if you try to run pytest with 32769 processes. I'd hazard a guess that it would anger the kernel gods and they'd immediately punish the puny user by killing all their processes.

Reply:
Excellent answer, thanks ! (indeed I must have been confusing processes and processors, I have shallow knowledge of parallelism, sometimes it shows :))